src/applications/check-in/api/versions/v2.js

```diff
@@ -74,10 +74,11 @@ const v2 = {
     };
   },
 
-  getCheckInData: async token => {
+  getCheckInData: async (token, reload = false) => {
     const url = '/check_in/v2/patient_check_ins/';
+    const typeSlug = reload ? '?reload=true' : '?reload=false';
```

Comment:
API_QUESTION: is there is difference between sending nothing and sending false?

Reply:
There is no difference for the API, I added these params so I could intercept them differently in cypress.